test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -508,17 +470,16 @@ public PassFailJFrame(String title, String instructions,
      * @see JTextArea#JTextArea(int,int) JTextArea(int rows, int columns)
      * @see Builder Builder
      */
+
     public PassFailJFrame(String title, String instructions,
                           long testTimeOut,
-                          int rows, int columns,
-                          boolean screenCapture)
+                          int rows, int columns)
             throws InterruptedException, InvocationTargetException {
         invokeOnEDT(() -> createUI(title, instructions,
                                    testTimeOut,
-                                   rows, columns,
-                                   screenCapture));
+                                   rows, columns));
     }
-
+    
```

Comment:
No trailing whitespace is allowed. The `jcheck` bot has marked it as an error and blocker.